src/main/java/edu/arizona/kra/protocol/onlinereview/dao/ProtocolOnlineReviewDaoOjb.java

```diff
@@ -0,0 +1,152 @@
+
+        if (matchingResultsCount == null || matchingResultsCount.intValue() <= searchResultsLimit) {
+            matchingResultsCount = new Long(0L);
+        }
+
+        // Below Code Adds Logic performed by filterresults()
+        Criteria criteria2 = criteria1.copy(true,true,true); /// Make a copy of the base criteria for reuse
+
+        /// Process Query1
+        criteria1.addLike("UPPER(reviewerApproved)", KRADConstants.YES_INDICATOR_VALUE);
+        criteria1.addLike("UPPER(adminAccepted)", KRADConstants.YES_INDICATOR_VALUE);
+        criteria1.addLike("UPPER(protocolOnlineReviewStatusCode)",  org.kuali.kra.irb.onlinereview.ProtocolOnlineReviewStatus.FINAL_STATUS_CD);
+
+        List protocolSubmissionStatusList1 = new ArrayList<String>(
+                Arrays.asList(ProtocolSubmissionStatus.APPROVED,
+                        ProtocolSubmissionStatus.EXEMPT,
+                        ProtocolSubmissionStatus.SPECIFIC_MINOR_REVISIONS_REQUIRED,
+                        ProtocolSubmissionStatus.SUBSTANTIVE_REVISIONS_REQUIRED,
+                        ProtocolSubmissionStatus.RETURNED_TO_PI));
+
+        Criteria subCriteria1 = new Criteria();
+        subCriteria1.addIn("submissionStatusCode",protocolSubmissionStatusList1);
+
+        ReportQueryByCriteria subQuery1 = QueryFactory.newReportQuery( ProtocolSubmission.class, subCriteria1);
+
+        // Just retrieve the submissionId from ProtocolSubmission (PROTOCOL_SUBMISSION .SUBMISSION_ID)
+        // since this is a subquery to find PROTOCOL_ONLN_RVWS.SUBMISSION_ID_FK IN
+        subQuery1.setAttributes(new String[] { "submissionId"});
+
+        criteria1.addIn("submissionIdFk", subQuery1);
+
+        QueryByCriteria porQuery1 = QueryFactory.newReportQuery( ProtocolOnlineReview.class, criteria1);
+
+        searchResults.addAll( getProtocolOnlineReviewsByQuery(porQuery1));
+
+        /// Process Query2
+        criteria2.addNotLike("UPPER(protocolOnlineReviewStatusCode)", org.kuali.kra.irb.onlinereview.ProtocolOnlineReviewStatus.FINAL_STATUS_CD);
+
+        Criteria subCriteria2 = new Criteria();
+
+        List protocolSubmissionStatusList2 = new ArrayList<String>(
+        Arrays.asList(ProtocolSubmissionStatus.IN_AGENDA,
+                ProtocolSubmissionStatus.SUBMITTED_TO_COMMITTEE));
+
+        subCriteria2.addIn("submissionStatusCode",protocolSubmissionStatusList2);
+        ReportQueryByCriteria subQuery2 = QueryFactory.newReportQuery( ProtocolSubmission.class, subCriteria2);
+
+        // Just retrieve the submissionId from ProtocolSubmission (PROTOCOL_SUBMISSION .SUBMISSION_ID)
+        // since this is a subquery to find PROTOCOL_ONLN_RVWS.SUBMISSION_ID_FK IN
+        subQuery2.setAttributes(new String[] { "submissionId"});
+
+        criteria2.addIn("submissionIdFk", subQuery2);
+
+        QueryByCriteria porQuery2 = QueryFactory.newReportQuery( ProtocolOnlineReview.class, criteria2);
+
+        searchResults.addAll( getProtocolOnlineReviewsByQuery(porQuery2));
```

Comment:
Set the actual result count so the system will display accurately the total number of results found.